modules/oe_bootstrap_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -235,4 +251,52 @@ public function toInternalLanguageId(string $language_code): string {
     return $language_code;
   }
 
+  /**
+   * Alter a link with BCL logic.
+   *
+   * @param \Twig\Environment $env
+   *   The env.
+   * @param string $label
+   *   The link text for the anchor tag as a translated string.
+   * @param \Drupal\Core\Url|string $path
+   *   The URL object or string used for the link.
+   * @param array|\Drupal\Core\Template\Attribute $attributes
+   *   An optional array or Attribute object of link attributes.
+   *
+   * @return array
+   *   The link render array.
+   */
+  public function bclLink(Environment $env, $label, $path, Attribute $attributes): array {
+    if (is_string($path) && !UrlHelper::isExternal($path)) {
```

Comment:
By looking at breadcrumb template/preprocess, they use directly markup to render the link. So we should go with the following: if the path is already a string, we return a rendered `<a>` element. We can use an inline template or return a render array 
```
return [
  '#type' => 'html_tag',
  '#tag'=> 'a',
  '#value' => $label,
  '#attributes' => $attributes,
];
```

Reply:
We should do the check in here because checking if a variable is string is not possible out of the box in twig.